docs/integrations.md

```diff
@@ -34,53 +34,58 @@ Zafira uses integration with Amazon S3 service to store user profile photos and
 * Grant read/write permissions for new user (read [detailed guide](https://aws.amazon.com/blogs/security/writing-iam-policies-how-to-grant-access-to-an-amazon-s3-bucket/))
 * Generate new access/secret keys
 * Turn on Amazon integration and provide access/secret keys and name of the bucket you've created
-* Press save and refresh the page, green light indicates correct integration status
+* Press save, green light indicates correct integration status
 
 <p>
   <img style="border: 1px solid grey;" width="280px" height="420px" src="../img/int_amazon.png">
 </p>
 
 
 #### Email
-Zafira provides functionality for sending test results, widgets and dashboards via email. You have to specify correct SMTP credentials to enable this feature. We are mostly using Gmail for that purposes. You can use configuration below replacing email and password with your valid Gmail credentials.
+Zafira provides functionality for sending test results, widget and dashboards via email. You have to specify correct SMTP credentials to enable this feature. We are mostly using Gmail for that purposes. You can use configuration below replacing email and password with your valid Gmail credentials.
```

Comment:
Zafira provides functionality for sending test results, widgets, and dashboards via email. You must provide correct SMTP credentials to enable this feature. We mainly use Gmail for these purposes. You can use the configuration below by replacing your email address and password with valid Gmail credentials.